examples/dynamoathenamigration/migration.go

```diff
@@ -404,6 +404,10 @@ func (t *task) fromS3ToChan(ctx context.Context, dataObj dataObjectInfo, eventsC
 	afterCheckpoint := afterCheckpointIn
 
 	scanner := bufio.NewScanner(gzipReader)
+	// By default maximum size of buffer for scanner is 60KB, which can be too
+	// low for dynamo entries which can be up to 400KB. Let's use 10MB as max.
```

Comment:
This comment is a little confusing. It says you need up to 400kB, so we'll use 10MB. Why 10MB when 400kB is enough? I'm guessing that part of that is to allow for gunzipping maybe, assuming 400kB max size is gzipped? But I'd rather not guess and have it spelled out for me how you arrived at 10MB.